src/i18n.js

```diff
@@ -8,15 +8,30 @@ const resources = {
   English: en,
 }
 
-i18n.use(initReactI18next).init({
-  resources,
-  fallbackLng: 'English',
-  interpolation: {
-    escapeValue: false,
-  },
-  react: {
-    useSuspense: true,
-  },
-})
+let fallbackLng = undefined
+if (navigator.language.includes('sv')) {
+  fallbackLng = 'Svenska'
+} else {
+  fallbackLng = 'English'
+}
+
```

Comment:
If short answer exists, what was the reason that you had to change both here and in useEffect?

Reply:
We did it for dropdown, as it wasn't showing correct language. But now as we are changing fallback language we dont' need it anymore. Removed it still works fine